lib/browser/guest-window-manager.js

```diff
@@ -321,6 +321,8 @@ handleMessage('ELECTRON_GUEST_WINDOW_MANAGER_WINDOW_POSTMESSAGE', (event, guestC
   // The W3C does not seem to have word on how postMessage should work when the
   // origins do not match, so we do not do |canAccessWindow| check here since
   // postMessage across origins is useful and not harmful.
+  assertCanAccessWindow(event.sender, guestContents, isRelatedWindow)
```

Comment:
comment is wrong now?

Reply:
@nornagon `assertCanAccessWindow` defaults to `canAccessWindow` for the check, I am passing `isRelatedWindow` here. Maybe I should rename `assertCanAccessWindow` to something else to avoid confusion?